debian/copyright

```diff
@@ -0,0 +1,620 @@
+                    GNU GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
+
+ Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
```

Comment:
This file should not contain the full license.

From the debian documentation:
```Packages distributed under the Apache license (version 2.0), the Artistic license, the GNU GPL (versions 1, 2, or 3), the GNU LGPL (versions 2, 2.1, or 3), and the GNU FDL (versions 1.2 or 1.3) should refer to the corresponding files under /usr/share/common-licenses, rather than quoting them in the copyright file.```

https://www.debian.org/doc/debian-policy/ch-docs.html#s-copyrightfile